src-tauri/src/widget_api/fs/mod.rs

```diff
@@ -0,0 +1,24 @@
+mod apis;
+pub mod utils;
+
+use tauri::{
+    plugin::{Builder, TauriPlugin},
+    Runtime,
+};
+
+// The registered apis can be invoked as "plugin:widget_api.fs|<api_name>"
```

Comment:
```suggestion
/// The registered APIs can be invoked as `"plugin:widget_api.fs|<api_name>"`.
```

NITS, and since we do call this function I think we can use docstring style.